forms.red

```diff
@@ -1,111 +1,152 @@
-WindowMinSize: 640x480
+WindowMinXSize: 800
+WindowMinYSize: 650
 
-; Toolboxes default values
-ToolboxDefXsize: 125
-ToolboxDefYsize: 300
-ToolboxBigSize: as-pair ToolboxDefXsize ToolboxDefYsize
-ToolboxMidSize: as-pair ToolboxDefXsize (ToolboxDefYsize / 1.5)
-ToolboxLowSize: as-pair ToolboxDefXsize (ToolboxDefYsize / 2)
-ToolboxWidgetList: ["area" "base" "box" "button" "calendar" "camera" "check" "drop-down" "drop-list" "field" 
-			"group-box" "image" "panel" "progress" "radio" "scroller" "slider" "tab-panel" "text" "text-list"]
-
-; Form sheet default values
 FormSheetDefXorigin: 145
-FormSheetDefYorigin: 10
+FormSheetDefYorigin: 17
 FormSheetDefOrigin: as-pair FormSheetDefXorigin FormSheetDefYorigin
-FormSheetDefXsize: WindowDefXsize - (ToolboxDefXsize * 2.3)
-FormSheetDefYsize: WindowDefYsize - 20
+FormSheetDefXsize: WindowDefXsize - FormSheetDefXorigin
+FormSheetDefYsize: WindowDefYsize - 135
 FormSheetDefSize: as-pair FormSheetDefXsize FormSheetDefYsize
 FormSheetStr: ""
 FormSheetCounter: 0
 FormSheetContent: []
 FormSheetRecodeBlock: []
-FormSheetWidgetSize: 300x250
-FormSheetWidgetBackground: orange
-FormSheetWidgetForeground: blue
+FormSheetWidgetSize: 200x150
+FormSheetWidgetBackground: gray
+FormSheetWidgetForeground: black
 
-; Font default values
-FontSel: attempt [make font! [name: "Consolas" size: 12 style: "normal" color:FormSheetWidgetForeground ] ]
-FontDefName: "Consolas"
-FontDefStyl: "Normal"
-FontDefSize: "12"
-
-; Editor default values
-EditorDefXsize: (FormSheetDefXsize / 2) - 5
-EditorDefYsize: (WindowDefYsize - FormSheetDefYsize) - 20
-EditorDefSize: as-pair EditorDefXsize EditorDefYsize
-StaticDefOrigin: as-pair FormSheetDefXorigin (FormSheetDefYorigin + FormSheetDefYsize + 5)
-DynamicDefOrigin: as-pair (FormSheetDefXorigin + EditorDefXsize + 5) (FormSheetDefYorigin + FormSheetDefYsize + 5)
+str: make string! FormSheetDefSize
+ToolboxDefXsize: 125
+ToolboxFileSize: as-pair ToolboxDefXsize 90
+ToolboxFontSize: as-pair ToolboxDefXsize 140
+ToolboxWidgetsSize: as-pair ToolboxDefXsize 200
+ToolboxFormSheetSize: as-pair ToolboxDefXsize 60
+ToolboxWidgetList: ["area" "base" "box" "button" "calendar" "camera" "check" "drop-down" "drop-list" "field" "group-box" "image" "panel" "progress" "radio" "scroller" "slider" "tab-panel" "text" "text-list"]
+
+DynamicEditorDefXsize: (WindowDefXsize / 3)
+StaticEditorDefXsize: (WindowDefXsize / 3 * 2)
+EditorDefYsize: 114
+DynamicEditorDefSize: as-pair DynamicEditorDefXsize EditorDefYsize
+staticEditorDefSize: as-pair StaticEditorDefXsize EditorDefYsize
+StaticDefOrigin: as-pair 0 (FormSheetDefYorigin + FormSheetDefYsize + 5)
+DynamicDefOrigin: as-pair StaticEditorDefXsize (FormSheetDefYorigin + FormSheetDefYsize + 5)
 DynamicCode: does copy [" "]
 
-; Database default values
-DbDefOrigin: as-pair (WindowDefXsize - (ToolboxDefXsize + 10)) FormSheetDefYorigin
-DbDefXsize: FormSheetDefXsize
-DbDefYsize: ToolboxDefYsize - 120
-DbDefSize: as-pair DbDefXsize DbDefYsize

```

Comment:
This code was here for preserving space as the I/O Branch would permit work with DB Servers, but I really don't care :-)